### PR TITLE
minor tweaks

### PR DESCRIPTION
**Pull request by BigMorty**
Update Readme and azure.yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,35 +45,51 @@ git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
 ```
 
 ### Create a virtual environment
-1) Open the terminal, navigate to the project folder, and run the following commands:
+Open the terminal, navigate to the project folder, and run the following commands:
 
+#### bash
```
Comment (ggailey777) [inline README.md:50]: I think that my original docs on this are incorrect and the tabs should be `Windows | Linux/macOs` but I'll fix this when update readmes (and in the regular Python quickstart too).

**Review by ggailey777 — APPROVED** [review]

**Review by paulyuk — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -50,19 +50,19 @@ Open the terminal, navigate to the project folder, and run the following command
...
 python -m venv .venv
-source .venv/scripts/activate
+source .venv/bin/activate
 ```
 
 #### PowerShell
 ```powershell
 py -m venv .venv
-.venv\scripts\activate
+.venv/scripts/activate
```
Comment (ggailey777) [inline README.md:59]: Both ways work on Windows PowerShell

File: README.md [hunk 3]
```diff
@@ -50,19 +50,19 @@ Open the terminal, navigate to the project folder, and run the following command
...
 ```
 
 #### Cmd
 ```cmd
 py -m venv .venv
-.venv\scripts\activate
+.venv/scripts/activate
```
Comment (ggailey777) [inline README.md:65]: This change doesn't work on Windows command prompt. Can you please revert it?

**Review by ggailey777 — CHANGES_REQUESTED** [review]
The change doesn't actually work on a Windows command prompt

File: README.md (hunk 3 above)
Comment (ggailey777) [reply README.md:65]: OK, I'll definitely be fixing our docs, which I think is the source of this confusion 🙄

**Review by paulyuk — COMMENTED** [review]
LGTM

**Pull request merged**